python/ray/services.py

```diff
@@ -1125,7 +1128,6 @@ def start_dashboard(require_webui,
     Returns:
         ProcessInfo for the process that was started.
     """
-    port = 8265  # Note: list(map(ord, "RAY")) == [82, 65, 89]
     while True:
```

Comment:
I guess you would like to maintain old behavior i.e. `port += 1` only when `port == ray_constants.DEFAULT_DASHBOARD_PORT`, or the documentation above needs update to:

> (Note that the port number increases if the ~default~ port is not available)

Reply:
Good point, thank you.